src/components/views/elements/TruncatedList.js

```diff
@@ -43,33 +53,49 @@ module.exports = React.createClass({
         };
     },
 
-    render: function() {
-        let childsJsx = this.props.children;
-        let overflowJsx;
-        const childArray = React.Children.toArray(this.props.children).filter((c) => {
-            return c != null;
-        });
+    _getChildren: function(min, max) {
```

Comment:
Would the names `start` and `end` not be more appropriate?

Reply:
(`slice` [uses](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice) `begin`, `end` )